### PR TITLE
feat: integrate tabs test

### PR DESCRIPTION
**Pull request by alexandereneroth**
Also fixed some layout inconsistencies / too large margins

### CODE REVIEW

File: src/app/pages/tabs-links-page/tabs-links-page.component.ts [hunk 1]
```diff
@@ -7,8 +9,17 @@ import { BreadcrumbsComponent } from 'src/app/components/breadcrumbs/breadcrumbs
   styleUrls: ['./tabs-links-page.component.css'],
   standalone: true,
   schemas: [CUSTOM_ELEMENTS_SCHEMA],
-  imports: [BreadcrumbsComponent],
+  imports: [BreadcrumbsComponent, RouterLink, RouterOutlet, CommonModule],
 })
 export class TabsLinksPageComponent {
+  fullPath: string;
 
+  constructor(private router: Router, private location: Location) {
```
Comment (gokberknur) [inline src/app/pages/tabs-links-page/tabs-links-page.component.ts:19]: It is usually good practice to shut it down the subscription. 

Even this component is destroyed (a.k.a detached from the DOM), subscription will stay active. 

there are different  ways to close a subscriptions. 

I am going to add re-usable method (in util file) to shut down a subscription when component is destroyed. You can check usage if you need it later :)

File: src/app/pages/tabs-subpage/tabs-subpage.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  standalone: true,
+  templateUrl: './tabs-subpage.component.html',
+  styleUrls: ['./tabs-subpage.component.css'],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+})
+export class TabsSubpageComponent {
+  currentRoute: string;
+
+  constructor(private route: ActivatedRoute) {
+    this.route.url.subscribe((segments) => {
```
Comment (gokberknur) [inline src/app/pages/tabs-subpage/tabs-subpage.component.ts]: above comment applies :) 

you can mark it as resolved when you read here :)  it is not a blocker.

File: src/app/pages/tabs-links-page/tabs-links-page.component.ts [hunk 3]
```diff
@@ -7,8 +9,17 @@ import { BreadcrumbsComponent } from 'src/app/components/breadcrumbs/breadcrumbs
...
+    this.router.events.subscribe(() => {
```
Comment (gokberknur) [inline src/app/pages/tabs-links-page/tabs-links-page.component.ts]: It is usually good practice to shut it down the subscription.

Even this component is destroyed (a.k.a detached from the DOM), subscription will stay active.

there are different ways to close a subscriptions.

I am going to add re-usable method (in util file) to shut down a subscription when component is destroyed. You can check usage if you need it later :)

**Review by gokberknur — APPROVED** [review]

File: src/app/pages/tabs-links-page/tabs-links-page.component.ts (hunk 3 above)
Comment (alexandereneroth) [reply src/app/pages/tabs-links-page/tabs-links-page.component.ts]: Thanks for reviewing!

**Pull request merged**